app/Components/FilterPanel/index.js

```diff
@@ -1,31 +1,62 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { sortMovies, filterMovies } from '../../actions';
 import './index.scss';
 
-const FilterPanel = ({movieList, movieListHook}) => 
+const FilterPanel = ({movies, sortMovies, filterMovies}) => 
     <div className='filterPanel'>
         <div className ='typeList'>
-            <a href = ''>ALL</a>
-            <a href = ''>DOCUMENTARY</a>
-            <a href = ''>COMEDY</a>
-            <a href = ''>HORROR</a>
-            <a href = ''>CRIME</a>
+            <a href = '#' onClick={(event) => {
+                event.preventDefault();
+                filterMovies('ALL');
+                }}>ALL</a>
+            <a href = '#' onClick={(event) => {
+                event.preventDefault();
+                filterMovies('DOCUMENTARY');
+                }}>DOCUMENTARY</a>
+            <a href = '#' onClick={(event) => {
+                event.preventDefault();
+                filterMovies('COMEDY');
+                }}>COMEDY</a>
+            <a href = '#' onClick={(event) => {
+                event.preventDefault();
+                filterMovies('HORROR');
+                }}>HORROR</a>
+            <a href = '#' onClick={(event) => {
+                event.preventDefault();
+                filterMovies('CRIME');
+                }}>CRIME</a>
```

Comment:
We can create a structure to store all options for filtering, implement handler and use 'useCallback' to prevent repeating of layout (DRY, KISS). 

To give a value for handler we can use smth like this: 
`{filters.map((filter) => 
    <span onClick={(filter) => filterHandler(filter) />`